portfolio/src/main/webapp/style.css

```diff
@@ -1,35 +1,53 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 700px;
 }
 
 .font {
   font-family: Optima, Arial, Helvetica, sans-serif;
 }
 
 .text-emphasis {
-    font-weight: bold;
+  font-weight: bold;
+}
+
+.navbar {
+  border-radius: 0 !important;
```

Comment:
See if you can get this style to apply without using `!important` by increasing the specificity of this selector. `!important` should be avoided wherever possible since it overrides all other selectors and even inline styles.

Reply:
Looks like it already has the expected behavior without using !important, so no need for it here